editor/pom.xml

```diff
@@ -67,5 +67,29 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Is this dependency needed?  If we weren't using it before, we don't seem to be now, either.

Reply:
yes without this it gives initializationError, java.lang.NoClassDefFoundError: org/hamcrest/SelfDescribing
Caused by: java.lang.ClassNotFoundException: org.hamcrest.SelfDescribing